ui/src/react-admin/modules/content-page/types/content-block.types.ts

```diff
@@ -385,6 +385,7 @@ export interface MediaGridBlockState extends DefaultContentBlockState {
 	ctaTitle?: string;
 	ctaTitleColor?: string;
 	ctaTitleSize?: HeadingType;
+	ctaTitleBackgroundColor?: string;
```

Comment:
shouldn't this property be used somewhere inside the component
or is that already the case, and were just now giving that existing property a value